maestro-cli/src/main/java/maestro/cli/device/DeviceService.kt

```diff
@@ -82,6 +81,14 @@ object DeviceService {
         val idbHost = "localhost"
         val idbPort = 10882
 
+        val isIdbCompanionRunning =
+            try { Socket(idbHost, idbPort).use { true } }
+            catch(_: Exception) { false }
+
+        if (isIdbCompanionRunning) {
```

Comment:
Should we perhaps reuse existing idb_companion instance if it is already running?

Reply:
I considered this possibility, and it'd probably work in the majority of cases.
Issues are:
- user might not notice that maestro is using their running idb, which can lead to surprises
- maestro asks the user to select a device, then continues to use an already started idb which is configured to use a different device

Reply:
That is a fair point